tables/he-IL-comp8.utb

```diff
@@ -1,18 +1,18 @@
-# liblouis: Multi-language computer braille table for Hebrew, Arabic and English (Israel).
+# liblouis: computer braille table for Hebrew (Israel).
 #
 #-index-name: Hebrew, Israel, computer
-#-display-name: Israeli computer braille
-#
-#+locale: mul-IL
+#-display-name: Hebrew computer braille
+#+locale: he
 #+type: computer
 #+dots: 8
 #
 #  Copyright (C) 1995-2009 by The BRLTTY Developers.
 #  Copyright (C) 2009 Samuel Thibault <samuel.thibault@ens-lyon.org>
 #  Copyright (C) 2012 Mesar Hameed <mesar.hameed@gmail.com>
-#  Copyright (C) 2014 Adi Kushnir <adikushnir@gmail.com>
```

Comment:
This line should not be removed and it should be "2014,2020 ..." and the last line should be removed.

Reply:
Fixed locally